pkgs/os-specific/linux/lvm2/common.nix

```diff
@@ -181,14 +181,17 @@ stdenv.mkDerivation rec {
     moveToOutput lib/libdevmapper.so $lib
   '';
 
-  passthru.tests = {
-    installer = nixosTests.installer.lvm;
-    lvm2 = recurseIntoAttrs nixosTests.lvm2;
-
-    # https://github.com/NixOS/nixpkgs/issues/369732
-    lvm2-fhs-env = buildFHSEnv {
-      name = "lvm2-fhs-env-test";
-      targetPkgs = p: [ p.lvm2 ];
+  passthru = {
+    features = { inherit (udev.passthru.features) withSystemd; };
```

Comment:
Similar to what I said for `umockdev`, I don't think every package consuming `libudev` needs to export this parameter.